tests/runner/src/utils/smoke/consume.rs

```diff
@@ -12,7 +12,7 @@ use fluvio_system_util::bin::get_fluvio;
 use fluvio::{Fluvio, Offset, PartitionConsumer};
```

Comment:
All smoke test related files should be toghether since they will be maintained as unit.  Why moved to separate `util` directory?